infra/partner-vm/resources.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  backend "gcs" {
+    bucket = "<your-tf-state-gcs-bucket>" # Example: terraform-multichain-state-bucket
+    prefix = "<your-multichain-prefix>" # Example: state/multichain-vm
+  }
+
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.73.0"
+    }
+  }
+}
+
+# These data blocks grab the values from your GCP secret manager, please adjust secret names as desired
+data "google_secret_manager_secret_version" "account_sk_secret_id" {
+  count   = length(var.node_configs)
+  secret  = "multichain-account-sk-dev-${count.index}"
```

Comment:
This actually forces the secret name to be multichain-account-sk-dev-x instead of giving partner freedom to choose their secret name? This is going to be problematic if the partner specifies some other name in node configs and store that into Secret manager.

Reply:
They can update this name to be whatever they choose, I just wanted something there to show the format, but I can use the variable in the tfvars file maybe, I'll give it a try

Reply:
Updated to use tfvars